src/v0/destinations/active_campaign/transform.js

```diff
@@ -337,7 +337,8 @@ const customListProcessor = async (message, category, destination, contactId) =>
   // status information
   // Ref: https://developers.activecampaign.com/reference/update-list-status-for-contact/
   const promises = [];
-  listArr.map(async (li) => {
+  // eslint-disable-next-line no-restricted-syntax
+  for (const li of listArr) {
```

Comment:
Instead of `for...of` loop, we can use map function to put things into promise
```javascript
const promises = listArr.map(async (li) => {
  return someVar;
})
.filter(v => undefined) // We would need to test this in order to understand how we remove undefined promises
```
This should be preferred approach

Reply:
I will take `no-restricted-syntax` in another PR, as changes introduced with new develop merge results these in different files. For now going with disabling the rule